Lib/test/libregrtest/run_workers.py

```diff
@@ -225,17 +225,20 @@ def create_stdout(self, stack: contextlib.ExitStack) -> TextIO:
     def create_json_file(self, stack: contextlib.ExitStack) -> tuple[JsonFile, TextIO | None]:
         """Create JSON file."""
 
-        json_file_use_filename = self.runtests.json_file_use_filename()
-        if json_file_use_filename:
-            # create an empty file to make the creation atomic
-            # (to prevent races with other worker threads)
-            prefix = TMP_PREFIX + 'json_'
-            json_fd, json_filename = tempfile.mkstemp(prefix=prefix)
-            os.close(json_fd)
-
-            stack.callback(os_helper.unlink, json_filename)
-            json_file = JsonFile(json_filename, JsonFileType.FILENAME)
+        json_file_use_stdout = self.runtests.json_file_use_stdout()
+        if json_file_use_stdout:
+            json_file = JsonFile(None, JsonFileType.STDOUT)
             json_tmpfile = None
+        #elif 0:
```

Comment:
Is the commented-out block on purpose?

Reply:
Yes. I spent time to implement this FILENAME code path, but it's no longer need. I hesitate to keep it.

Later, it may be interesting to use it on Emscripten/WASI platforms. But for that, we should first find a way to expose a temporary directory "shared" between the Linux x86-64 Python and the Emscripten/WASI Python.

Reply:
hum, since @AA-Turner reported another issue, I changed my mind and I just removed it :-)